libs/langchain-google-cloud-sql-pg/src/chatMessageHistory.ts

```diff
@@ -0,0 +1,86 @@
+import { BaseChatMessageHistory } from "@langchain/core/chat_history";
+import { BaseMessage } from "@langchain/core/messages";
+import PostgresEngine from "./engine.js";
+
+export class PostgresChatMessageHistory extends BaseChatMessageHistory {
+  lc_namespace: string[];
+
+  engine: PostgresEngine;
+  
+  sessionId: string;
+  
+  tableName: string;
+  
+  schemaName: string;
+
+  constructor(engine: PostgresEngine, sessionId: string, tableName: string, schemaName: string = "public") {
```

Comment:
It looks like a lot of the other integrations use an Inputs interface for the constructor, example https://github.com/langchain-ai/langchainjs/blob/0f9139d95f1f73e3cd260141f86c0e3ffc68de3d/libs/langchain-mongodb/src/chat_history.ts. Let's follow that pattern and turn the constructor input into a object